dataprofiler/profilers/profile_builder.py

```diff
@@ -2290,7 +2295,11 @@ def _update_null_replication_metrics(self, clean_samples: Dict) -> None:
             sum_not_null = np.delete(total_row_sum, col_id) - sum_null
 
             mean_null = sum_null / null_count
-            mean_not_null = sum_not_null / true_count
+
+            if true_count == 0:
+                mean_not_null = sum_not_null
```

Comment:
technically this isn't a `mean` value ... its a `summation` value, though?

Reply:
i'd suggest doing a:
```python
mean_not_null = np.nan
 if true_count:
    mean_not_null = sum_not_null / true_count
```

Reply:
+1 @tony